src/main/java/hudson/tasks/junit/SuiteResult.java

```diff
@@ -348,4 +349,19 @@ void setParent(hudson.tasks.junit.TestResult parent) {
     private static final long serialVersionUID = 1L;
 
     private static final Pattern SUREFIRE_FILENAME = Pattern.compile("TEST-(.+)\\.xml");
+
+    /**
+     * Merges another SuiteResult into this one.
+     * 
+     * @param sr
+     */
+    public void merge(SuiteResult sr) {
+        if (hasTimeAttr) {
```

Comment:
yes that should be hasTimeAttr .. this is the closest currently which rely on the previously mentionet - all of them has it or neither - logic....